charts/seed-bootstrap/templates/fluentd-es/fluentd-es-ds.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluentd-es
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: fluentd-es
+    role: logging
+---
+kind: ClusterRole
+apiVersion: {{ include "rbacversion" . }}
+metadata:
+  name: fluentd-es
+  labels:
+    app: fluentd-es
+    role: logging
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "namespaces"
```

Comment:
remove quotes for all resources and verbs. They are not needed.

Reply:
Fixed.